extensions/python/src/main/resources/jet_to_python_init.sh

```diff
@@ -5,8 +5,10 @@ set -e -x
 
 python3 -m venv jet_to_python_env --system-site-packages
 source jet_to_python_env/bin/activate
-python3 -m pip install --upgrade pip
-python3 -m pip install protobuf==@protobuf.version@ grpcio==@grpc.version@
+
+flock --exclusive /tmp/jet-pip.lock python3 -m pip install --upgrade pip
```

Comment:
We should use another directory. The `/tmp` is shared between the host and containers on our Jenkins.